public/src/main/java/com/revenuecat/purchases/LogLevel.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases
+
+enum class LogLevel {
+    VERBOSE,
+    DEBUG,
+    INFO,
+    WARN,
+    ERROR;
+
+    /**
+     * Whether debug logs are enabled
+     */
+    val debugLogsEnabled: Boolean
```

Comment:
Hmm this is accessible by devs... should we make this and the method below `internal`? I don't think it would help devs to have access to these.

Reply:
I made this `internal`, but the other one I had to leave `public` because it's used from 2 different packages.

Reply:
Ahh right 🤦 ... I really want to try to unify these modules soon. 

What comes to mind is moving these 2 to the `common` module as extensions properties/functions. Maybe to the `logUtils.kt` file?
```
val LogLevel.debugLogsEnabled: Boolean
    get() = this <= LogLevel.DEBUG

fun LogLevel.Companion.debugLogsEnabled(enabled: Boolean): LogLevel {
    return if (enabled) {
        LogLevel.DEBUG
    } else {
        LogLevel.INFO
    }
}
```

Might need fixing some imports but they should be accessible to both the `common` and `purchases` modules once they are there and not be accessible to devs.

Reply:
That works thanks 👍🏻